### ISSUE TITLE
Adjust paths after repository rename

### ISSUE THREAD

**Issue by lunedis** [jvalue/ods#354]
In `.env` we define 
```
DOCKER_REGISTRY=docker.pkg.github.com/jvalue/open-data-service
```

Do we need to / should we change that, now that the repository is called *jvalue/ods*?

All `package.json` files also still refer to the old repository url, e.g. here: https://github.com/jvalue/ods/blob/main/ui/package.json#L50

And of course the CI badge in the `README.md`.

**Comment by georg-schwarz** [comment]
Agreed, we should change it. Not sure if it even works as it is now...

**Comment by lunedis** [comment]
CI: https://github.com/jvalue/ods/blob/main/.github/workflows/ods.yml#L346
